src/font.ts

```diff
@@ -373,22 +373,29 @@ export class Font {
   }
 
   /**
-   * Load the web fonts that are used by ChordSymbol. For example, `flow.html` calls:
+   * Load the web fonts that are used by your app.
+   * If fontNames is undefined, all fonts in Font.WEB_FONT_FILES will be loaded.
+   *
+   * For example, `flow.html` calls:
    *   `await Vex.Flow.Font.loadWebFonts();`
    * Alternatively, you may load web fonts with a stylesheet link (e.g., from Google Fonts),
    * and a @font-face { font-family: ... } rule in your CSS.
```

Comment:
hi @ronyeh,
the approach I like most for the future is to use font-face statements. I did that in the inspection tool `glyphs.html`. But I do not know how to force an await on css. do you?